.swiftlint.yml

```diff
@@ -7,8 +7,6 @@ analyzer_rules:
 
 disabled_rules:
   - trailing_whitespace
-  - todo
```

Comment:
Why enable the todo rule? I think we will never get rid of all the todos in our repo.

Reply:
We can re-disable this later. I'm leaving it in for now because we've got a lot of todos that we actually want to tackle before merging mqtt to main.

Reply:
I am fine with that, however, this will just create noise in every PR. We should just look for todos in the final PR when reviewing.
<img width="862" alt="image" src="https://github.com/awslabs/aws-crt-swift/assets/7613045/cb548488-00b8-489d-99e3-1c0d4fb26eb1">